examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-ingress
+---  
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-ingress-serviceaccount
+  namespace: nginx-ingress
+---  
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: nginx-ingress-clusterrole
+rules:
+  - apiGroups:
+      - ""
+      - "extensions"
+    resources:
+      - configmaps
+      - secrets
```

Comment:
Yes I can retrieve the secret by name but I need to know then a secret is updated or removed (and that is the reason of the use of informers)